skdecide/hub/domain/gym/gym.py

```diff
@@ -1060,14 +1072,14 @@ def _get_goals_(self):
         )
 
 
-class AsGymEnv(gym.Env):
-    """This class wraps a scikit-decide domain as an OpenAI Gym environment.
+class AsLegacyGymV21Env(LegacyEnv):
```

Comment:
I like this solution of providing both AsLegacyGymV21Env & AsGymnasiumEnv (based on LegacyEnv & EnvCompatibility) very much, because it provides highest flexibility for end-users and still only requires Gymnasium dependency! Just a warning: the gymnasium.wrappers.compatibility module (present in Gymnasium until latest 0.29.1 official release) seems to have disappeared on the main branch :( Not sur why (and maybe not a big deal for now), but may be worth investigating for the future...

Reply:
OK perfect, I agree with you